File: src/app/core/models/asset.ts
export class Asset {
    asset_class: string;
    initial_allocation: number;
    asset_mgmt_fee: number;
    initial_load_fee: number;

    constructor(assetClass: string) {
        this.asset_class = assetClass;
        this.initial_allocation = 1000;
        this.asset_mgmt_fee = 0.0015;
        this.initial_load_fee = 0;
    }
}

File: Simulation/Simulation.Domain/Entities/Asset.cs
using Newtonsoft.Json;

namespace Simulation.Domain.Entities
{
    public class Asset
    {
        [JsonProperty("asset_class")]
        public string AssetClass { get; set; }
        [JsonProperty("initial_allocation")]
        public double InitialAllocation { get; set; }
        [JsonProperty("asset_mgmt_fee")]
        public double AssetMgmtFee { get; set; }
        [JsonProperty("initial_load_fee")]
        public double InitialLoadFee { get; set; }

        public Asset() { }
    }
}
